src/interactive/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+<future release version placeholder>
```

Comment:
0.6.0

Reply:
This PR targets a feature branch. It doesn't mean or need to be released in 0.6.0. It can be any future version when the feature is ready. So I use the placeholder to make it stand out in the future when we merge the feature branch to the dev branch. Can we make the change at that time?

Reply:
OK, make sense!